common/src/main/java/com/wynntils/models/characterstats/CharacterStatsModel.java

```diff
@@ -98,7 +98,7 @@ private int getMaxSoulPoints() {
      */
     private int getCurrentSoulPoints() {
         ItemStack soulPoints = McUtils.inventory().getItem(8);
-        if (soulPoints.getItem() == Items.NETHER_STAR || soulPoints.getItem() == Items.DIAMOND_AXE) {
+        if (soulPoints.getItem() == (Models.Character.isHunted() ? Items.DIAMOND_AXE : Items.NETHER_STAR)) {
```

Comment:
I think this change is kind of unnecessary. (Especially if you do the change I request to `isHunted`, then it becomes completely tautological.) 

In fact, we already know and trust that slot 8 is the soul points, so this is more of a sanity check/assertion. Had it been written today with the style we have now (and not copied verbatim from Legacy), it would have looked like:
```
 private int getCurrentSoulPoints() {
        ItemStack soulPoints = McUtils.inventory().getItem(8);
        if (soulPoints.getItem() != Items.NETHER_STAR &&  soulPoints.getItem() != Items.DIAMOND_AXE) return -1;

        return soulPoints.getCount();     
}
```